apps/sms/js/thread_ui.js

```diff
@@ -2049,6 +2049,12 @@ var ThreadUI = {
       case 'contextmenu':
         evt.preventDefault();
         evt.stopPropagation();
+
+        // Disable contextmenu while selecting non-editable message content
+        if (!this.threadMessages.classList.contains('editable-select-mode')) {
```

Comment:
There seems no default "contextmenu" event app-wise. If we don't handle contextmenu and preventDefault the event, it would be the click event for the element.